assignments/prototypes.js

```diff
@@ -41,7 +41,39 @@
 
 // Test you work by un-commenting these 3 objects and the list of console logs below:
 
-/*
+function GameObject(attrs) {
+  this.createArt = attrs.createArt;
+  this.name = attrs.name;
+  this.dimensions = attrs.dimensions;
+};
+  GameObject.prototype.destroy = function(){
+    return `${this.name} was removed from the game.`;
+};
+
+function CharacterStats(stats) {
+  GameObject.call(this,stats);
+  this.healthPoints = stats.healthPoints;
+};
+CharacterStats.prototype = Object.create(GameObject.prototype);
+CharacterStats.prototype.takeDamage = function(){
+  return `${this.name} took damage.`;
+};
+
+function Humanoid(humanAttrs){
+  CharacterStats.call(this, humanAttrs);
+  GameObject.call(this, humanAttrs);
```

Comment:
ok, so, here I see you call both CharacterStats and GameObject, this is unecessary, since CharacterStats calls GameObject into it, you are essentially calling GameObject when you call CharacterStats, making line `64` obsolete